docs/home.md

```diff
@@ -50,3 +49,10 @@ go get -u github.com/checkr/flagr
 cd $GOPATH/src/github.com/checkr/flagr
 make all
 ```
+
+If you're modifying the UI, you want may to enable automatic build-on-change:
+
+```
+cd browser/flagr-ui/
+npm run watch
```

Comment:
I think you can also mention that 
- `npm run serve` if people want to run a vue dev server to just serve the frontend. 
- `npm run watch` if people want to see UI changes from flagr 18000 port - just like the production build.

Reply:
is there a recommended way that you've used for dev to work on the UI? i can just document that and simplify it if its complicated?

hoping to keep it as simple and accessible as your `make run` helper

huge +1000 for makefiles as they're great standard helpers to get things goin :)

Reply:
how about a new make command:

```
watch_ui:                                     
    @echo "Watch ./browser/flagr-ui for UI development ..."
    @cd ./browser/flagr-ui/; npm run watch                                  
```

Reply:
@zhouzhuojie sorry - was more asking around serve. when you use that do you just access the UI on a different port?

Reply:
I was thinking that `npm run build --watch` was slow, turns out the build only takes about 3s, so I think we can recommend people to use the build with watch, which is closer to the production experience - e.g. you open localhost:18000 and hit browser refresh. 

But there is some magic from vue-cli serve, like hot reload for css and js changes, so if people prefer, they can also run `npm run serve`.

Reply:
I'm going to suggest a small change to this in an hour or so when i get some cycles.

Reply:
I added a top-level npm package which uses a simple library to run multiple processes, it means you can just `npm start` and you'll get the frontend (in serve mode) and backend running.

Reply:
I learned that `make` supports parallel commands:

```
run:
	@$(PWD)/flagr --port 18000

run_ui:
	@cd ./browser/flagr-ui/; npm run serve

start:
	$(MAKE) -j run run_ui
```

and then just do `make run` starts both.

Reply:
ah good call thats easier